apps/api/src/app/change/e2e/promote-changes.e2e.ts

```diff
@@ -5,12 +5,16 @@ import { expect } from 'chai';
 import { CreateNotificationTemplateDto } from '../../notification-template/dto/create-notification-template.dto';
 import { UpdateNotificationTemplateDto } from '../../notification-template/dto/update-notification-template.dto';
 import { MessageTemplateRepository } from '../../../../../../libs/dal/src/repositories/message-template/message-template.repository';
+import { NotificationGroupRepository } from '../../../../../../libs/dal/src/repositories/notification-group/notification-group.repository';
+import { EnvironmentRepository } from '../../../../../../libs/dal/src/repositories/environment/environment.repository';
 
 describe('Promote changes', () => {
   let session: UserSession;
   const changeRepository: ChangeRepository = new ChangeRepository();
   const notificationTemplateRepository: NotificationTemplateRepository = new NotificationTemplateRepository();
   const messageTemplateRepository: MessageTemplateRepository = new MessageTemplateRepository();
+  const notificationGroupRepository: NotificationGroupRepository = new NotificationGroupRepository();
+  const environmentRepository: EnvironmentRepository = new EnvironmentRepository();
```

Comment:
```suggestion
  const notificationGroupRepository = new NotificationGroupRepository();
  const environmentRepository = new EnvironmentRepository();
```

Not critical but, TS infers this automatically so no really need to specify it like this :)